src/node.rs

```diff
@@ -600,8 +612,8 @@ impl PbftNode {
         let mut peer_res = Ok(());
         if let Some((msg, sender_id)) = self.msg_log.pop_backlog() {
             debug!(
-                "{}: Popping from backlog {} from {:?}",
-                state, msg.message_type, sender_id
+                "{}: Popping message from {:?} from backlog",
```

Comment:
Should be `Popping message {:?} from backlog`

Reply:
The `{:?}` here is the sender ID, so it should end up looking like `Popping message from $SENDERID from backlog`, which seems correct (or at least readable) to me

Reply:
My bad!